Fixing issues in verification case 1c (#121)
idaholab/TMAP8: doc/content/verification_and_validation/ver-1c.md

TMAP4 performs comparison of the mobile species concentration at three points - (a) a point at the interface (x = 0 m), (b) a point at the end of the pre-loaded region (x = 10 m), and (c) a point in the initally unloaded region (x = 12 m). The comparison of the values calculated through TMAP8 and analytically for the TMAP4 case is shown in
[ver-1c_comparison_time_TMAP4]. TMAP7 performs comparison of the mobile species concentration at the same points (b) and (c) as in the TMAP4 case, but performs the third comparison at (d) a point close to the interface (x = 0.25 m). The comparison of the values calculated through TMAP8 and analytically for the TMAP7 case is shown in
[ver-1c_comparison_time_TMAP7]. The TMAP8 calculations are found to be in good agreement with the analytical solution.

!media figures/ver-1c_comparison_time_TMAP4.png
    style=width:50%;margin-bottom:2%
    id=ver-1c_comparison_time_TMAP4

>     caption=Comparison of concentration as function of time at x\=0 m, 10 m and 12 m

Reviewer: ```suggestion
    caption=Comparison of concentration as a function of time at x\=0 m, 10 m, and 12 m
```

>     calculated through TMAP8 and analytically (TMAP4 case)

Reviewer: ```suggestion
    calculated with TMAP8 and analytically (TMAP4 cases)
```

!media figures/ver-1c_comparison_time_TMAP7.png
    style=width:50%;margin-bottom:2%
    id=ver-1c_comparison_time_TMAP7

>     caption=Comparison of concentration as function of time at x\=0.25 m, 10 m and 12 m
>     calculated through TMAP8 and analytically (TMAP7 case)

Reviewer: ```suggestion
    caption=Comparison of concentration as a function of time at x\=0.25 m, 10 m, and 12 m
    calculated with TMAP8 and analytically (TMAP7 cases)
```

# Diffusion Problem with Partially Preloaded Slab

This verification problem is taken from [!cite](longhurst1992verification,ambrosek2008verification). Diffusion of tritium through a semi-infinite SiC layer is modeled with an initial

> loading of 1 atom/m$^3$ in the first 10 m of a 100 m slab. TMAP4 uses a slab length of 2275 m, however using a smaller slab length was found to not change the results. Additionally, the smaller domain size allows getting a finer simulation mesh for the same computational cost, which improves the agreement between the TMAP8 and analytical calculations.

Reviewer: ```suggestion
loading of 1 atom/m$^3$ in the first 10 m of a 100 m slab. TMAP4 and TMAP7 cases use a slab length of 2275 m, however using a smaller slab length was found to not change the results. Additionally, the smaller domain size allows getting a finer simulation mesh for the same computational cost, which improves the agreement between the TMAP8 and analytical calculations.
```

Author: I thought about this, but TMAP7 did not actually mention a slab length so I wanted to wait till I see the TMAP7 fortran scripts to be sure.

Reviewer: ^ I updated the suggestion.

Diffusivity is set to 1 m$^2$/s
and no traping is included. The analytical solution is given by:

\begin{equation}
\label{eq:c_func}
C = \frac{C_0}{2} \left[ \text{erf}\bigg(\frac{h-x}{2\sqrt{Dt}}\bigg) +
\text{erf}\bigg(\frac{h+x}{2\sqrt{Dt}}\bigg)  \right]
\end{equation}

where $h=10$ m is the thickness of the pre-loaded portion of the layer.

> !alert warning title=Typo in [!cite](longhurst1992verification)

Reviewer: ```suggestion
!alert warning title=Typo in [!cite](longhurst1992verification,ambrosek2008verification)
```
